Revamp 'Merchants' page (and remove list of merchants)

erciccione opened the pull request:
Closes #1736

The list of merchants have been removed and replaced with links to external merchants directories. We also provide a list of p2p, non-custodial exchanges.

I did an extensive research to find all non custodial and P2P exchanges, but sadly i ended up with a list of only two elements: Bisq and HodlHodl. Please let me know if there are other exchanges with those characteristics, so we can add them.

Feel free to suggest better wording or changes. ~~Opening as draft for now.~~

Preview:
![Screenshot 2021-08-24 at 12-30-38  Merchants Exchanges ](https://user-images.githubusercontent.com/28106476/131101333-32a18268-a3dc-43dc-836b-e9c00dbc6780.png)

AiloQito commented:
I would advise to specify a link to the cryptocurrency Monero in the cotalog.
https://cryptwerk.com/pay-with/xmr/

AiloQito commented:
I would advise to specify a link to the cryptocurrency Monero in the cotalog.
https://cryptwerk.com/pay-with/xmr/

erciccione commented:
>  https://cryptwerk.com/pay-with/xmr/

we already point to that url.

johnr365 commented on _i18n/en.yml:
```diff
@@ -126,12 +126,19 @@ hangouts:
-  intro3: (select the "Merchant" template and fill up all sections).
+  descr: On this page we list non-custodial P2P exchanges and links to merchants directories.
+  useit: "Use Monero!"
+  use: During the years merchants of all kinds have come to value the financial privacy that Monero brings. In fact, merchants adoption is continuously rising.
```
```suggestion
  use: During the years merchants of all kinds have come to value the financial privacy that Monero brings. In fact, merchants adoption is continuously rising.
```
use: Over the years, merchants of all kinds have come to value the financial privacy that Monero brings. In fact, merchant adoption is continuously rising.

erciccione replied:
Changed.

johnr365 commented on _i18n/en.yml:
```diff
@@ -126,12 +126,19 @@ hangouts:
+  descr: On this page we list non-custodial P2P exchanges and links to merchants directories.
+  useit: "Use Monero!"
+  use: During the years merchants of all kinds have come to value the financial privacy that Monero brings. In fact, merchants adoption is continuously rising.
+  dirdescr: "If you are looking for places where to spend or exchange your Monero coins, take a look at these directories of merchants, but remember to make your own researches before trusting them with your money:"
```
```suggestion
dirdescr: "If you are looking for places to spend or exchange your Monero, the directories below have some options. Please remember to do your own research before trusting them with your Monero:"
```

erciccione replied:
Changed, except the last word. I prefer to keep "money" instead of "Monero"

erciccione replied:
Changed, except the last word. I prefer to keep "money" instead of "Monero"

selsta commented on _i18n/en.yml:
```diff
@@ -126,12 +126,19 @@ hangouts:
+  cardonion: "Onion address:"
+  hodlhodldescr: P2P trading platform with non-custodial escrow.
   disclaimer: |
     "Please note: these links are being provided as a convenience and for informational purposes only; they do not constitute an endorsement by the Monero community of any products, services or opinions of the corporations or organizations or individuals listed. The Monero community bears no responsibility for the accuracy, legality, or content of these external sites. Contact the external site f ...
```
Unrelated but there `" "` show up on the page, is it intentional?

erciccione replied:
I don't know to be honest, but could be.

SamsungGalaxyPlayer commented:
LocalMonero isn't included?

SamsungGalaxyPlayer commented:
I think that removing all the other exchanges substantially reduces the utility of the page. Even if they're in a separate section, links to inform people that Monero trades on exchanges like Kraken, Binance, ChangeNOW, etc., and that people can purchase gift cards using something like CoinCards is useful information.

selsta commented:
Thinking about this again, yea agree with @SamsungGalaxyPlayer, it's surprising to see HodlHodl linked that does not seem to have any Monero offers?

We should support Monero focused platforms like LocalMonero.

SamsungGalaxyPlayer commented:
I understand that LocalMonero's online trade option can be considered custodial in some senses, but they also facilitate in-person P2P buys/sells for example.

erciccione commented:
> I think that removing all the other exchanges substantially reduces the utility of the page

The point of these proposed changes is to not have to deal with exchanges anymore. Arbitrarely readding some exchanges that support Monero doesn't make sense IMO and would make this change pointless.

> I understand that LocalMonero's online trade option can be considered custodial in some senses

Localmonero, to my surprise, is fully custodial, i don't know what you mean with "in some senses", but you are not in control of your coins and you have to 100% trust the operators, also in case of cash <-> xmr exchanges. If we want to list only non custodial and P2P exchanges (and i think we should), that's the list.

> We should support Monero focused platforms like LocalMonero.

I agree. The problem is that there are none. We should push localmonero to provide keys of user's wallets instead.

selsta commented:
We are not going to break the chicken egg problem by sending new users to a website where they can't even buy any Monero. It's just a bad user experience.

There are no rules that forces us to list 2 "exchanges" only (there is enough space for a third one) and there is also no rule that custodial exchanges aren't allowed to be listed.

So IMO if you want to list HodlHodl then that's fine but LocalMonero should also be listed. That's all I will say to this issue.

erciccione commented:
> We are not going to break the chicken egg problem by sending new users to a website where they can't even buy any Monero

My point is not to make people buy Monero as fast as possible to fill their bags. IMO on Getmonero we should suggest people non-custodial exchanges, because those are safer and really make people in control of their coins. We provide two links of merchant directories where they can find all the centralized exchanges they want.

> there is also no rule that custodial exchanges aren't allowed to be listed.

Of course not, how could that be possible? That's what i'm proposing right now and that was discussed in the issue and on #monero-site. The idea was to add only non custodial and P2P exchanges, after looking deeply i found only two exchanges with those characteristics. Listing only two wasn't planned, it was a sad surprise.

> So IMO if you want to list HodlHodl then that's fine but LocalMonero should also be listed. That's all I will say to this issue.

I don't see the benefit of adding P2P non-custodial exchanges + LocalMonero. Sound like a recipe for troubles, since if we make an exception, everybody will want us to make more.

A compromise i see is could be to list P2P exchanges, so also custodial ones. That's a step back from my proposal, but would include LocalMonero in the list.

selsta commented:
> I don't see the benefit of adding P2P non-custodial exchanges + LocalMonero. Sound like a recipe for troubles, since if we make an exception, everybody will want us to make more.

I think that would be fine because the chart could make it clear if the exchange is custodial or not.

SamsungGalaxyPlayer commented:
I believe this is unnecessarily restrictive based on morals that gets in the way of most user's interests on the page. Just pick the major non-scam custodial ones by volume. Also as mentioned, LocalMonero allows in-person p2p trades in addition to online trades.

The proposed page isn't actionable to the point that it makes a messy page into a mostly useless page.

erciccione commented:
> I think that would be fine because the chart could make it clear if the exchange is custodial or not.

@selsta yes, i will add all respectable P2P exchanges i'm aware of and will specify in the description if they are custodial or not. 

> I believe this is unnecessarily restrictive based on morals

@SamsungGalaxyPlayer It's important to educate Getmonero's users to using P2P (preferably non-custodial) exchanges. Centralized exchanges do not give control of the coins to their users and they are a major point of centralization, which is dangerous for the whole Monero network since they can be shut down, hacked, confiscated or made illegal from one day to another. Decentralized/P2P exchanges are always to be preferred because they are much more resistant to censorship and empower the individual. Of course, all this doesn't matter if the goal is only to onboard as many people as possible without caring of their safety and the long term stability of the network; but that's not my goal.

People can find lists of centralized exchanges supporting Monero literally everywhere (since most people suggest those) and we link to 2 merchants directories where more can be found. Your only point seems to be that you want centralized exchanges to have the most visibility on this page and that's not going to happen in my proposal for the reasons i already explained.

Doesn't make much sense to use slogans like "your keys, your coin", but then when it comes to take actual steps to empower people by educating them about the existence of exchanges that give them their keys, you prefer to suggest centralized exchanges, with no motivation other than "that's what people use". I think we can and must do better than that.

> Just pick the major non-scam custodial ones by volume

I already explained why giving the most visibility to centralized exchanges is bad, but this specific suggestion is naive, arbitrary and would bring us back to the original problem. Beside, I cannot think of a worse metric than "by volume" when presenting an exchange. We are better than that.

erciccione commented:
- Changed description from "non-custodial P2P exchanges" to only "P2P exchanges" and added "non-custodial" entry in the cards
- Added LocalMonero and SimpleSwap
- Added buttons at the bottom instead of links (it's nicer IMO)
- Updated screenshots

Now, as mentioned in my last comment, the only thing left is to add a link to a directory of centralized exchanges and if that's not available, a list of some centralized exchanges, listed by reputation. This should be the case only if nothing else is possible.

SamsungGalaxyPlayer commented:
I am confused about the custodial definition then. I think what you're trying to get at is whether there is counterparty risk.

SinpleSwap is an example of an instant exchanger that doesn't hold an account for you. This is similar to many others, like ChangeNOW, Sideshift AI, etc. So it's somewhat correct to say it's noncustodial, but I think a user may imply this also means there is no counterparty risk. They can still steal your funds since you send funds to a wallet they have the keys to. This is the main differentiator between traditional instant exchangers and atomic swap exchangers.

SimpleSwap doesn't support BTC/USD.

selsta reviewed (commented):
LocalMonero has an onion address: nehdddktmhvqklsnkjqcbpmb63htee2iznpcbs5tgzctipxykpj6yrid.onion/

erciccione commented:
> SinpleSwap is an example of an instant exchanger that doesn't hold an account for you. 

I got mislead by the wording on their website. They do seems to be a normal instant exchange. Will remove.

> SimpleSwap doesn't support BTC/USD.

They claim to allow people to by cryptocurrencies with fiat (https://simpleswap.io/faq#fiat-to-crypto-exchanges)

> LocalMonero has an onion address: nehdddktmhvqklsnkjqcbpmb63htee2iznpcbs5tgzctipxykpj6yrid.onion

I was under the impression they had one but then ended up removing it. I could be confusing them with something else. Will change.

erciccione commented:
Added changes and updated screenshot.

erciccione commented:
I haven't found a directory of exchanges supporting Monero. Is anybody aware of any? Otherwise we have to figure out a way to add centralized exchanges to this page without going back to the original problem (#1736).

HoverHalver commented:
https://kycnot.me/ is offline, 
but there is a mirror at https://patchysicp.xyz/~wheezy/kycnotme/

erciccione commented:
> mirror at https://patchysicp.xyz/~wheezy/kycnotme/

Some info here appear wrong. Like HodlHodl shown as not supporting Monero, but they do support it.

erciccione commented:
I'm thinking of adding a "centralized exchanges" box at the bottom that would be divided in 2 sections: Exchanges and Swappers.

To not go back to the original issue i think we have two solutions:

1. We add a short list of exchanges we know are reputable and we don't give the possibility to add new ones like we do now. The Getmonero team will take care of adding new items to the list if they that makes sense.
2. We find an external list of exchanges supporting Monero and we link to that. If something like that doesn't exist, might even make sense to ask around if anybody is willing to create and maintain such list.

Please let me know what are your thoughts about that. If we don't find a solution soon we could go ahead and merge the merchants list as suggested in this PR and take care of centralized exchanges later on.

erciccione commented:
- Added list of CEXes and swappers
- Added mention of Atomic Swaps
- Screenshot updated

Ready for reviews.

Riolinov commented:
I would advise to add this rating https://bitpro.cash/en/
https://bitpro.cash/en/XMR-to-BTC
https://bitpro.cash/en/XMR-to-ETH
I sometimes select exchangers here when I can not find a suitable one on your site 
This rating works for more than a year and it always includes only verified exchangers and very convenient to see who has the minimum amount and exchange rate

erciccione commented:
Thanks @Riolinov, but i never heard of this service and unless somebody can vouch for it i'm inclined to not add it to the page.

rating89us commented on _i18n/en.yml:
```diff
@@ -127,12 +127,32 @@ hangouts:
+  use: Over the years, merchants of all kinds have come to value the financial privacy that Monero brings. In fact, merchant adoption is continuously rising.
+  dirdescr: "If you are looking for places to spend your Monero, the directories below have some options. Please remember to do your own research before trusting them with your money:"
+  descrp2p: The best way to exchange Monero for other currencies (or the other way round) is using P2P exchanges, because they allow traders to directly interact with each other, without the need of trusted third parties (except maybe for providing an escrow service). These exchanges are more private then their centralized counterparts and allow people to be in complete control of their coins if  ...
+  swapsdescr: A new exciting way to exchange Monero is Atomic Swaps, which <a href="LINK">we announced</a> on August 2021. Atomic Swaps are a completely decentralized way to swap Bitcoin <-> Monero (more info in the blog post). The cechnology is brand new, so there aren't many swap providers yet, but you can already sell XMR for BTC using this protocol at
```
```suggestion
  swapsdescr: A new exciting way to exchange Monero is Atomic Swaps, which <a href="LINK">we announced</a> on August 2021. Atomic Swaps are a completely decentralized way to swap Bitcoin <-> Monero (more info in the blog post). The technology is brand new, so there aren't many swap providers yet, but you can already sell XMR for BTC using this protocol at
```

rating89us commented on _i18n/en.yml:
```diff
@@ -127,12 +127,32 @@ hangouts:
+  visitlocalmonero: Visit LocalMonero
+  visithodlhodl: Visit HodlHodl
+  centrexchanges: Centralized exchanges
+  centrexchangesp: If you prefer to use centralized echanges, here is a list of renown CEXes and swappers. Many more exchanges support Monero, we list here only few reputable ones.
```
```suggestion
  centrexchangesp: If you prefer to use centralized exchanges, here is a list of renowned CEXes and swappers. Many more exchanges support Monero, we list here only a few reputable ones.
```

rating89us commented on _i18n/en.yml:
```diff
@@ -127,12 +127,32 @@ hangouts:
+  useit: "Use Monero!"
+  use: Over the years, merchants of all kinds have come to value the financial privacy that Monero brings. In fact, merchant adoption is continuously rising.
+  dirdescr: "If you are looking for places to spend your Monero, the directories below have some options. Please remember to do your own research before trusting them with your money:"
+  descrp2p: The best way to exchange Monero for other currencies (or the other way round) is using P2P exchanges, because they allow traders to directly interact with each other, without the need of trusted third parties (except maybe for providing an escrow service). These exchanges are more private then their centralized counterparts and allow people to be in complete control of their coins if  ...
```
```suggestion
  descrp2p: The best way to exchange Monero for other currencies (or the other way round) is using P2P exchanges, because they allow traders to directly interact with each other, without the need of trusted third parties (except maybe for providing an escrow service). These exchanges are more private then their centralized counterparts and allow people to be in complete control of their coins if they are non-custodial, but they usually have less volume than 'traditional' exchanges. In the section below, we offer a list of P2P exchanges that support Monero.
```

rating89us commented on _i18n/en.yml:
```diff
@@ -127,12 +127,32 @@ hangouts:
+  use: Over the years, merchants of all kinds have come to value the financial privacy that Monero brings. In fact, merchant adoption is continuously rising.
+  dirdescr: "If you are looking for places to spend your Monero, the directories below have some options. Please remember to do your own research before trusting them with your money:"
+  descrp2p: The best way to exchange Monero for other currencies (or the other way round) is using P2P exchanges, because they allow traders to directly interact with each other, without the need of trusted third parties (except maybe for providing an escrow service). These exchanges are more private then their centralized counterparts and allow people to be in complete control of their coins if  ...
+  swapsdescr: A new exciting way to exchange Monero is Atomic Swaps, which <a href="LINK">we announced</a> on August 2021. Atomic Swaps are a completely decentralized way to swap Bitcoin <-> Monero (more info in the blog post). The technology is brand new, so there aren't many swap providers yet, but you can already sell XMR for BTC using this protocol at
```
missing link

rating89us commented on _i18n/en.yml:
```diff
@@ -30,7 +30,7 @@ titles:
   hangouts: Hangouts
   sponsorships: Sponsorships
-  merchants: Merchants & Services
+  merchants: Merchants & Exchanges
```
I would also change the menu item (line 55 of this file) to `Merchants & Exchanges`

erciccione replied:
That would make it too long to fit on our menu bars i'm afraid.

rating89us replied:
It's not really a problem because multiple languages already have menu items using two lines:
![image](https://user-images.githubusercontent.com/45968869/131464930-a8d86647-f43d-44b0-8f90-551a187b664a.png)

erciccione replied:
Some languages do have too long menu items and in some sections it's a problem. Personally here i don't care. This is how it 
would look like:
![MENU](https://user-images.githubusercontent.com/28106476/131466251-4adeb890-d4d9-4271-999a-e2867c837014.png)

It breaks the format at least on my screen, by being the only entry on two line (in all dropdown menus), but if people are ok with that i'll just change it.

rating89us replied:
Yes, this is the only menu item in English version that has a line break, but I don't see this as a problem since all other languages have at least one menu item with a line break.

erciccione replied:
Agreed. Clarity is more important than consistency in this case.

rating89us commented on _i18n/en.yml:
```diff
@@ -127,12 +127,32 @@ hangouts:
-  intro2:  open a GitHub issue and let us know
-  intro3: (select the "Merchant" template and fill up all sections).
+  descr: On this page we list merchant directories, where you can spend your Moneroj and P2P, decentralized and centralized exchanges.
+  useit: "Use Monero!"
```
This "Use Monero!" title is more like a slogan, it says nothing about the section content

rating89us commented on _i18n/en.yml:
```diff
@@ -127,12 +127,32 @@ hangouts:
-  intro1: Merchants of all kinds have come to value the financial privacy that Monero brings. Below is a list of the merchants that we know of that currently accept Monero for their goods and services. <b>These merchants are not vetted by the Getmonero team, see the disclaimer at the bottom of this page</b>. If a company no longer accepts Monero or you would like your business to be listed, pleas ...
-  intro2:  open a GitHub issue and let us know
-  intro3: (select the "Merchant" template and fill up all sections).
+  descr: On this page we list merchant directories, where you can spend your Moneroj and P2P, decentralized and centralized exchanges.
```
```suggestion
  descr: On this page you will find stores and merchants that accept Monero as payment, as well as exchanges that trade Monero for other currencies.
```

rating89us replied:
A less technical description that explains what exchanges are used for

erciccione replied:
I want to point out that we post both centralized and decentralized exchanges. We can remove `P2P` if that's seen too technical.

rating89us replied:
P2P is too technical, as well as the centralized x decentralized differentiation.

I don't think it's needed to list the different types of exchanges on the first sentence.

I think it's better to first give a simple explanation of what an exchange is/what it is used for. Maybe for us it seems obvious, but many new users don't know what an "exchange" is, and we don't want to throw 3 more unknown terms for these users on the first sentence.

rating89us commented on _i18n/en.yml:
```diff
@@ -127,12 +127,32 @@ hangouts:
+  visitbisq: Visit Bisq
+  visitlocalmonero: Visit LocalMonero
+  visithodlhodl: Visit HodlHodl
+  centrexchanges: Centralized exchanges
```
```suggestion
  centrexchanges: Centralized exchanges (CEXs) & Swappers
```

rating89us commented on community/merchants/index.md:
```diff
@@ -4,74 +4,110 @@ title: titles.merchants
+        <ul class="logo">
+            <li><a href="https://www.kraken.com/">Kraken</a> (EUR, USD, CAD, GBP, JPY, AUD, CHF)</li>
+            <li><a href="https://www.binance.com/trade.html?symbol=XMR_BTC">Binance</a> (USD, EUR, RUB, TRY, NGN, UAH, KZT, INR, ...)</li>
+            <li><a href="https://dvchain.co/">DV Chain</a> (OTC) (USD, CAD, GBP, EUR, JPY, ...)</li>
```
```suggestion
            <li><a href="https://dvchain.co/">DV Chain (OTC)</a> (USD, CAD, GBP, EUR, JPY, ...)</li>
```

rating89us commented:
The anchor on this home page link is not working:
![image](https://user-images.githubusercontent.com/45968869/131624660-4989d378-7e6a-4fd5-80a4-a7025db6e0bd.png)

erciccione commented:
> The anchor on this home page link is not working:

Seems to be working fine for me.

rating89us commented:
> Seems to be working fine for me.

I mean the home page link points to `/community/merchants/#exchanges`, but there is no #exchanges anchor on the Merchants & Exchanges page. It opens the Merchants & Exchanges scrolled to the top (displaying merchants section first).

erciccione commented:
@rating89us got it now. Fixed.

rating89us commented on community/merchants/index.md:
```diff
@@ -4,74 +4,110 @@ title: titles.merchants
+        <h3>CEXes</h3>
+        <p>{% t merchants.cexp %}</p>
+        <ul class="logo">
+            <li><a href="https://www.kraken.com/">Kraken</a> (EUR, USD, CAD, GBP, JPY, AUD, CHF)</li>
+            <li><a href="https://www.binance.com/trade.html?symbol=XMR_BTC">Binance</a> (USD, EUR, RUB, TRY, NGN, UAH, KZT, INR, ...)</li>
+            <li><a href="https://dvchain.co/">DV Chain (OTC)</a> (USD, CAD, GBP, EUR, JPY, ...)</li>
+            <li><a href="https://www.bitfinex.com/">Bitfinex</a> (USD)</li>
+          </ul>
```
```suggestion
            <li><a href="https://www.kraken.com/">Kraken</a> (EUR*, USD*, CAD, GBP, JPY, AUD, CHF)</li>
            <li><a href="https://www.binance.com/trade.html?symbol=XMR_BTC">Binance</a> (USD, EUR, RUB, TRY, NGN, UAH, KZT, INR, ...)</li>
            <li><a href="https://dvchain.co/">DV Chain (OTC)</a> (USD*, CAD*, GBP*, EUR*, JPY*, ...)</li>
            <li><a href="https://www.bitfinex.com/">Bitfinex</a> (USD*)</li>
          </ul>
          <p>*Fiat currency to Monero trading pair (e.g. USD/XMR, EUR/XMR)</p>
```

rating89us replied:
What if we add which of these fiat currencies have direct trading pair with Monero?

erciccione replied:
added

LocalMoneroScam commented:
> Thinking about this again, yea agree with @SamsungGalaxyPlayer, it's surprising to see HodlHodl linked that does not seem to have any Monero offers?
> 
> We should support Monero focused platforms like LocalMonero.

you should not include LocalMonero, Localmonero is scam check this [LocalMonero Scam Admin issue]https://github.com/monero-project/monero-site/issues/1889(url) for evidence
